slide/src/test/common.rs

```diff
@@ -43,7 +43,7 @@ macro_rules! printer {
 macro_rules! fail {
     ($report:expr) => {
         libtest_mimic::Outcome::Failed {
-            msg: Some(atomic_lock($report)),
+            msg: Some(std::sync::Arc::new(std::sync::Mutex::new($report))),
```

Comment:
so youre making an arc to disallow mutating and the using a mutex to mutate? I'm confused by this

Reply:
No, this is an atomic mutex. So we can pass the mutex between threads.